module/zfs/fm.c

```diff
@@ -158,7 +158,7 @@ zfs_zevent_drain(zevent_t *ev)
 }
 
 void
-zfs_zevent_drain_all(int *count)
+zfs_zevent_drain_all(uint32_t *count)
```

Comment:
Why is it uint32_t here while zevent_len_cur is uint_t.

Reply:
It is an artifact of my treatment of `uint32_t` as a short hand for `unsigned int`. I will change it.